RELEASES.md

```diff
@@ -2,6 +2,34 @@
 
 > Note: ES3/IE8 compatibility will be removed in the future v3.x.x releases (scheduled for mid-late 2022), so if you need to retain ES3 compatibility you will need to remain on the 2.x.x versions of the SDK or your runtime will need install polyfill's to your ES3 environment before loading / initializing the SDK.
 
+## Unreleased Changes
+
+### Potential behavioral changes
+
+This release enhances the cookie management behavior when cookies are disabled. Previously, when cookies were disabled, calls to `cookieMgr.set()` would return `false` and cookie values would be lost. Now, these operations are cached in memory and automatically applied when cookies are re-enabled.
+
+**Behavior changes:**
+- `cookieMgr.set()` now returns `true` when cookies are disabled (because values are cached), instead of `false`
+- `cookieMgr.get()` now returns cached values when cookies are disabled, instead of empty strings
+- `cookieMgr.del()` operations are now cached and applied when cookies are re-enabled
+- Applications can now recover cookie state after temporary cookie blocking scenarios
+
+**These changes improve data persistence and are considered enhancements rather than breaking changes.** If your application logic depends on the previous behavior of `set()` returning `false` when cookies are disabled, you may need to check `cookieMgr.isEnabled()` instead.
```

Comment:
Lets add a new "disableCaching" to the ICookieMgrConfig and default to false, add this config hear as a way to maintain the previous behavior. Add additional caching tests that validate that the caching is disabled and the previous behavior still occurs.